mooncake-store/tests/eviction_strategy_test.cpp

```diff
@@ -0,0 +1,134 @@
+// eviction_strategy_test.cpp
+#include <glog/logging.h>
+#include <gtest/gtest.h>
+
+#include <memory>
+
+#include "eviction_strategy.h"
+
+namespace mooncake {
+
+// Test fixture for EvictionStrategy tests
+class EvictionStrategyTest : public ::testing::Test {
```

Comment:
Great unit test! We should also make sure it behaves correctly once it's integrated into master. Would you mind adding a simple integration test as well?